app/views/products/_product.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="col-md-4 product-men">
+  <div class="men-pro-item simpleCart_shelfItem">
+    <div class="men-thumb-item">
+      <%= image_tag product.images.first&.image if product.images.first&.image %>
+      <div class="men-cart-pro">
+        <div class="inner-men-cart-pro">
+          <%= link_to t("quick_view"), "/detail", class: "link-product-add-cart" %>
+        </div>
+      </div>
+      <span class="product-new-top">New</span>
+    </div>
+    <div class="item-info-product">
+      <h4><a href="single.html"><%= product.name %></a></h4>
+      <div class="info-product-price">
+        <span class="item_price"><%= number_to_currency(product.price, negative_format: "(%u%n)") %></span>
+      </div>
+      <div
+        class="snipcart-details top_brand_home_details item_add single-item hvr-outline-out button2"
+      >
+        <form action="#" method="post">
+          <fieldset>
+            <input type="hidden" name="cmd" value="_cart" />
+            <input type="hidden" name="add" value="1" />
+            <input type="hidden" name="business" value=" " />
+            <input
+              type="hidden"
+              name="item_name"
+              value="Party Men's Blazer"
+            />
+            <input type="hidden" name="amount" value="30.99" />
+            <input type="hidden" name="discount_amount" value="1.00" />
+            <input type="hidden" name="currency_code" value="USD" />
+            <input type="hidden" name="return" value=" " />
+            <input type="hidden" name="cancel_return" value=" " />
+            <input
+              type="submit"
+              name="submit"
+              value="Add to cart"
+              class="button"
+            />
```

Comment:
form gì đây nhỉ??

Reply:
form add to cart, b Hoa làm phần này nên e để tạm đó ạ